docs/accessibility/configuration/views.mdx

```diff
@@ -2,7 +2,7 @@
 title: 'Views of URLs and mounted components | Cypress Accessibility'
 description: 'Cypress Accessibility automatically groups certain URL patterns to create views. For URLs that are not automatically grouped, the views property allows you to specify your own URL patterns that represent views.'
 sidebar_label: views
-sidebar_position: 40
+sidebar_position: 10
```

Comment:
I'm not understand the ordering of these pages at all - it feels like the config pages should be alphabetized

Reply:
Previously it seemed more disorganized, I didn't spot that it was maybe alphabetical by default without explicit positions? When the list is short, it's hard to tell that it's alphabetical vs just random order. 

The reordering was intended to make more sense as a logical reading order. Like if you are actually doing config or learning about it, making the views make sense and filtering unwanted views is useful before you even think about elements. Removing unwanted elements and setting up significant attributes makes sense before you set up attribute filters.